src/components/Spinner/Spinner.styles.ts

```diff
@@ -0,0 +1,44 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { addAlpha, getTheme } from '../../util/colors';
+import { mergeStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+import { SpinnerColor, SpinnerSize } from './types';
+import { getGutterValue } from '../../util/styles/gutters';
+
+export interface SpinnerClassNames {
+  root: string;
+  label: string;
+}
+
+const labelMargins = {
+  [SpinnerSize.XSMALL]: `-1px 0 0 ${getGutterValue(1.5, true)}`,
```

Comment:
Gutters are not part of the theme, so we assume they won't change, right?

Reply:
yes